.circleci/config.yml

```diff
@@ -14,11 +14,14 @@ workflows:
               only: /v?[0-9]+(\.[0-9]+)*/
           matrix:
             parameters:
-              python_version: ["2.7", "3.5"]
-              django_version: ["django111", "django22"]
+              python_version: ["3.5", "3.8"]
+              debian_version: ["stretch", "buster"]
```

Comment:
why this?

Reply:
that is the debian version, previous these changes the version was stretch by default and that version only supports until python 3.5  hence it's required a new debian version which supports python 3.8, besides there is no a image called 3.8-stretch  in docker hub